src/components/AddFoodForm.jsx

```diff
@@ -0,0 +1,93 @@
+import { Form, Input, InputNumber, Button } from 'antd';
+
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+function AddFoodForm({ addNewFood }) {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [calories, setCalories] = useState(0);
+  const [servings, setServings] = useState(1);
+
+  const handleNameInput = (e) => {
+    setName(e.target.value);
+  };
+
+  const handleImageInput = (e) => {
+    setImage(e.target.value);
+  };
+
+  // ok, this got me confused... somehow the InputNumber-Component from AntDesign
```

Comment:
this is something that can be expected though. 

What I am trying to say is, that it's a component, who knows what's going on inside it, just because it has the same name as the input element from HTML doesn't mean that it's going to work the same way as onChange. If in doubt, look at the [docs](https://ant.design/components/input-number/#:~:text=%2D-,onChange,function(value%3A%20number%20%7C%20string%20%7C%20null),-%2D).